packages/spec/integration/graphql/__tests__/develop.js

```diff
@@ -1,4 +1,4 @@
-describe('graphql developmet server', () => {
+describe('graphql development client', () => {
```

Comment:
This meant that the test starts the development server, therefore `graphql development server` is actually correct ;)